portals/application/controllers/KlearCustomRunCodeController.php

```diff
@@ -77,8 +78,7 @@ protected function _runCode($type, $inputMac, $error = false){
                 $this->view->terminalModel = $terminalModelsModel;
 
                 $var = base64_encode(serialize($this->view->getVars()));
-                $command = "/usr/bin/php ".$route . DIRECTORY_SEPARATOR .$filename." ".$var." 2>&1"; 
-
+                $command = "/usr/bin/php ".$route . DIRECTORY_SEPARATOR .$filename." ".$var." 2>&1";
```

Comment:
Not sure what changed in this line :thinking: 
The contatenation spaces are random, as it seems everywhere in this code.
But don't worry, we'll fix that in a future (never).